app/renderer/js/main.ts

```diff
@@ -743,6 +743,30 @@ class ServerManagerView {
 						});
 					}
 				},
+				{
+					label: (DomainUtil.getDomain(index).muted ? 'Unmute' : 'Mute'),
+					click: () => {
+						dialog.showMessageBox({
+							type: 'warning',
+							buttons: ['YES', 'NO'],
+							defaultId: 0,
+							message: 'Are you sure you want to ' + (DomainUtil.getDomain(index).muted ? 'unmute' : 'mute') + ' this organization?'
+						}, response => {
+							if (response === 0) {
+								if (DomainUtil.getDomain(index).muted) {
+									// server is already muted
+									this.tabs[index].webview.updateBadgeCount(DomainUtil.getDomain(index).badgeCount);
+									DomainUtil.updateBadgeCount(index, -1);
+									DomainUtil.updateMute(index, false);
+								} else {
+									DomainUtil.updateBadgeCount(index, this.tabs[index].webview.badgeCount);
+									DomainUtil.updateMute(index, true);
+									this.tabs[index].webview.updateBadgeCount(0);
```

Comment:
Why do we update the `badgeCount` to 0 here?

Reply:
Here the badge-count in the webview has been updated to zero, in order to hide the badge from the organization.